react-redux/counter.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+// import { createStore } from 'redux';
```

Comment:
use redux ```createStore```. don't write it yourself. you can comment your implementation of ```createStore``` for your purpose